osquery/tables/system/linux/shadow.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <shadow.h>
+
+#include <boost/regex.hpp>
```

Comment:
For includes, we try to stick with a broadest -> narrowest listing, so third party includes come after C and C++ libs. Would you mind bringing this down below the `#include <mutex>`?